tendrl/node_agent/monitoring/collectd/collectors/gluster/heavy_weight/tendrl_gluster_heal_info.py

```diff
@@ -0,0 +1,168 @@
+import collectd
+import sys
+import traceback
+
+sys.path.append('/usr/lib64/collectd/gluster')
```

Comment:
> is this to fix the utils import issue?

Yes

> you can put these utils in the node-agent code, and simply do "from tendrl.node_agent.monitoring.collectd import utils"

The util is specific more to gluster-integration than node-agent..
But implementation of the util is in accordance with requirements from graphite like encoding status into integer and so on.. which are very much so only because graphite or any other time-series data is supposed only to be non-string integral values. So I refrained from putting it in gluster-integration also..

Anyway once we move to the new clean approach of the gevent based plugin framework within the collectd, we will be writing off this temporary sys path modifications